src/index.ts

```diff
@@ -65,7 +65,12 @@ function transformer(
 		if (transformed) {
 			code = applySourceMap(transformed, filePath);
 		}
-	} else {
+	} else if (
+		// Best guesses for files that need to be transformed.
+		code.includes('{export ') ||
+		code.includes('{import ') ||
+		code.split('\n').some((line) => line.startsWith('import ') || line.startsWith('export '))
```

Comment:
We could bypass this check (and always transformSync) when the extension is not `.js`.